setup.py

```diff
@@ -70,24 +80,24 @@ class bdist_wheel_half_pure(bdist_wheel):
         """Create OS-dependent, but Python-independent wheels."""
 
         def get_tag(self):
-            pythons = 'py2.py3.' + PYTHON_INTERPRETERS
+            pythons = 'py3.' + '.'.join(PYTHON_INTERPRETERS)
             if platform == 'darwin':
-                oses = MACOSX_VERSIONS
+                oses = '.'.join(MACOSX_VERSIONS)
             elif platform == 'win32':
                 if architecture0 == '32bit':
                     oses = 'win32'
                 else:
                     oses = 'win_amd64'
             else:
-                pythons = 'py2.py3'
+                pythons = 'py3'
                 oses = 'any'
             return pythons, 'none', oses
 
     cmdclass['bdist_wheel'] = bdist_wheel_half_pure
 
 setup(
     name='soundfile',
-    version='0.10.3post1',
+    version='0.10.3-post1',
```

Comment:
If I'm reading PEP 440 right, it should be 0.10.3.post1. https://peps.python.org/pep-0440/

Reply:
Yeah, you're right. For [pre-release-separators](https://peps.python.org/pep-0440/#pre-release-separators):
> Pre-releases should allow a ., -, or _ separator between the release segment and the pre-release segment. The normal form for this is without a separator

So both work, but without the `-` is more correct. That's an unfortunate partial incompatibility with semver.